.github/workflows/check-package.yml

```diff
@@ -11,20 +11,31 @@ on:
         description: 'PyPI name to install the package with'
         required: true
         type: string
+      strategy-matrix:
```

Comment:
let's add also exclude and include... fine with adding it in another PR :rabbit:

Reply:
Actually, it is already possible with `strategy-matrix` as `matrix` contains `include` inside:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixinclude